### ISSUE TITLE
how to customize dropdown arrow icons

### ISSUE THREAD

**Issue by softerboy** [EducationLink/vue-tel-input#113]

**Comment by softerboy** [comment]
Code example:

```html
<vue-tel-input v-model="phone">
  <template 
    slot="arrow-icon"
    slot-scope="{ open }">
      <i v-if="open" class="fa fa-arrow-up"></i>
      <i v-else class="fa fa-arrow-down"></i>
  </template>
</vue-tel-input>
```

**Comment by iamstevendao** [comment]
Fixed in [v2.4.3](https://github.com/EducationLink/vue-tel-input/releases/tag/v2.4.3).
Thanks @softerboy!

**Comment by Nehemiah01** [comment]
Tried this, it's not working for me, along with some other props as well e.g (dropdownOptions.showSearchBox).
Here's my code:

`<vue-tel-input
                            v-model="form.phoneNumber" 
                            name="phoneNumber"
                            @keypress="handleKeyPress"
                            @blur="handleBlur"
                            :inputOptions="{
                                placeholder: 'Phone Number',
                                required: 'true',
                                autocomplete: 'off',
                                id: 'phone',
                                styleClasses: {'border border-red-500': (phoneBlurred && phoneEmpty) || (phoneBlurred && !phoneValid) } /*styleClasses*/
                            }"
                            :dropdownOptions="{
                                showSearchBox: true,
                                showFlags: true,
                                showDialCodeInList: false
                            }"
                            mode="international"
                            :validCharactersOnly="true"
                        >
                            <template 
                            slot="arrow-icon"
                            slot-scope="{ open }">
                                <img v-if="open" src="@/assets/down-arrow.svg" alt="">
                                <img v-else class="rotate-180" src="@/assets/down-arrow.svg" alt="">
                            </template>
                        
                        </vue-tel-input>`
